src/library/trackcollectionmanager.cpp

```diff
@@ -17,6 +19,8 @@ const QString kConfigGroup = QStringLiteral("[TrackCollection]");
 
 const ConfigKey kConfigKeyRepairDatabaseOnNextRestart(kConfigGroup, "RepairDatabaseOnNextRestart");
 
+constexpr int kThreadPoolTimeoutMillisOnShutdown = 30000; // 30 sec
```

Comment:
Is 30 seconds just an arbitrary value? How long would be a "usual" amount of time that this would take?